### PR TITLE
Run benchmarks in separate threads

### PR DESCRIPTION
**Pull request by ivan-tymoshenko**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)
     
Снимок экрана 2022-05-23 в 15 23 23

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -5,7 +5,10 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "benchmark": "node bench.js",
+    "bench": "node ./benchmark/bench.js",
+    "bench:cmp": "node ./benchmark/bench-cmp-branch.js",
+    "bench:cmp:ci": "node ./benchmark/bench-cmp-branch.js --ci",
+    "benchmark": "node ./benchmark/bench-cmp-lib.js",
```
Comment (mcollina) [inline package.json:11]: It's missing the devDeps changes

Comment (ivan-tymoshenko) [reply package.json:11]: Sorry, I didn't get you. What should be here?

File: benchmark/bench-cmp-branch.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const { spawn } = require('child_process')
+
+const chalk = require('chalk')
+const inquirer = require('inquirer')
+const simpleGit = require('simple-git')
```
Comment (mcollina) [inline benchmark/bench-cmp-branch.js:7]: I think those dependencies needs to be added to devDeps

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**